app.py

```diff
@@ -0,0 +1,95 @@
+from flask import Flask, request, send_file
+from flask_limiter import Limiter
+from PIL import Image, ImageOps
+import numpy as np
+
+import tensorflow as tf
+import tensorflow.keras
+
+import numpy as np
+import PIL.Image
+import io
+import tensorflow_hub as hub
+
+hub_module = hub.load('https://tfhub.dev/google/magenta/arbitrary-image-stylization-v1-256/2')
+
+def load_img(img):
+  byteIO = io.BytesIO()
+  img.save(byteIO, format='PNG')
```

Comment:
Do we only get the PNG format?

Reply:
No, but we need to specify the format before transfer the image. we can get png, jpg and jpeg.